Tutorials/Editors-Manual/Getting-Started-With-Umbraco/Umbraco-Interface/index.md

```diff
@@ -1,28 +1,58 @@
 ---
-versionFrom: 7.0.0
-needsV8Update: "true"
+versionFrom: 8.0.0
 ---
 
 # Umbraco Interface
 
 ## Initial View
 
-Once you have logged in you will enter the Content section of Umbraco. There are several sections within Umbraco that allow you to manage different aspects of your site. The Content section allows you to manage the web pages of your site. Sections may vary from user to user based upon their role. As an editor or writer for instance you may only have access to the Content and Media sections. You can navigate between sections using the black section bar on the left-hand side of the screen. As you rollover one of the sections the name of the section will be revealed. Click a section to navigate to it.
+After logging in to an Umbraco V8 project you will be presented with a screen with a lot of different buttons in this guide you will gain a quick overview of what each button does.
```

Comment:
Took the liberty to split this into two sentences.
Also, we prefer writing `Umbraco 8` instead of `V8` 😄 

```suggestion
After logging in to an Umbraco 8 project you will be presented with a screen with a lot of different buttons. In this guide you will gain a quick overview of what each button does.
```

Reply:
done